plugin.yaml

```diff
@@ -51,6 +57,9 @@ dsl_definitions:
         # On Ansible 2.8.x "INVALID_TASK_ATTRIBUTE_FAILED" default value has
         # been changed to "True" which cause failure when run playbook
         ANSIBLE_INVALID_TASK_ATTRIBUTE_FAILED: "False"
+        # On Ansible 2.8 if the user want to change the default python module
+        # he can use the following "INTERPRETER_PYTHON" config to override that
+        INTERPRETER_PYTHON: "/usr/bin/python"
```

Comment:
@ahmadiesa-abu Do not forget to update the docs about these changes 

CC @EarthmanT